CHANGELOG.md

```diff
@@ -13,6 +13,79 @@
+scope when requesting an id_token from Dex
+(PR [#2529](https://github.com/scality/metalk8s/pull/2529))
+
+- [#2443](https://github.com/scality/metalk8s/issues/2443) - Improve error
+handling for salt jobs in the UI
+(PR [#2475](https://github.com/scality/metalk8s/pull/2475))
+
+- [#2495](https://github.com/scality/metalk8s/issues/2495) - Fix monitoring
+page to display all alerts
+(PR [#2503](https://github.com/scality/metalk8s/pull/2503))
+
+### Enhancements
+- [#1461](https://github.com/scality/metalk8s/issues/1461) - Improve JSON
+parsing in storage-operator
+(PR [#2577](https://github.com/scality/metalk8s/pull/2577))
+
+- [#2422](https://github.com/scality/metalk8s/issues/2422) - Replace
+config.yaml + product.txt with a single Solutions manifest.yaml
+(PR [#2565](https://github.com/scality/metalk8s/pull/2565))
+
+- [#2423](https://github.com/scality/metalk8s/issues/2423) - Upgrade
+nginx-ingress-controller to version 0.30.0
+(PR [#2446](https://github.com/scality/metalk8s/pull/2446))
+
+- [#2429](https://github.com/scality/metalk8s/issues/2429) - Upgrade Dex
+to version 2.23.0
+(PR [#2437](https://github.com/scality/metalk8s/pull/2437))
+
+- [#2430](https://github.com/scality/metalk8s/issues/2430) - Upgrade
+prometheus-operator version to 8.13.0
+(PR [#2557](https://github.com/scality/metalk8s/pull/2557))
+
+- [#2431](https://github.com/scality/metalk8s/issues/2431) - Upgrade
+Prometheus-adapter addon to version 0.6.0
+(PR [#2441](https://github.com/scality/metalk8s/pull/2441))
+
+- [#2569](https://github.com/scality/metalk8s/issues/2569) - Add digest for
+dex config
+(PR [#2573](https://github.com/scality/metalk8s/pull/2573))
+
+- [#2579](https://github.com/scality/metalk8s/issues/2579) - Add pre-upgrade
+& pre-downgrade states to cleanup Prometheus-operator-grafana Deployments
+(PR [#2580](https://github.com/scality/metalk8s/pull/2580))
+
+- [#2488](https://github.com/scality/metalk8s/issues/2488) - Add ability to
+update CSC defaults during upgrade/downgrade
+(PR [#2513](https://github.com/scality/metalk8s/pull/2513))
+
+- [#2493](https://github.com/scality/metalk8s/issues/2493) - Use async call
+for disk.dump during Volume provisioning
+(PR [#2571](https://github.com/scality/metalk8s/pull/2571))
+
+- [#2590](https://github.com/scality/metalk8s/issues/2590) - Bump Kubernetes
+version to 1.16.10 (PR [#2597](https://github.com/scality/metalk8s/pull/2597))
+
+- Add support for CustomResourceDefinition apiextensions.k8s.io/v1
+(PR [#2583](https://github.com/scality/metalk8s/pull/2583))
+
+### Documentation
+- [#2488](https://github.com/scality/metalk8s/issues/2488) - Rework cluster
+and service configuration docs to make defaults upgrade-able
+(PR [#2501](https://github.com/scality/metalk8s/pull/2501))
+
+- [#2522](https://github.com/scality/metalk8s/issues/2522) - Expose default
+service configurations in the Operational guide section
+(PR [#2532](https://github.com/scality/metalk8s/pull/2532))
```

Comment:
Do we want to talk about documentation changes in the changelog ?? @gdemonet @thomasdanan @NicolasT

Reply:
Removed!

Reply:
Maybe later, when we build changelogs from snippets.